libqtile/hook.py

```diff
@@ -87,6 +87,24 @@ def startup_complete(self, func):
         """
         return self._subscribe("startup_complete", func)
 
+    def shutdown(self, func):
+        """Called when qtile is shutdown
```

Comment:
Maybe change "when" to "before"? That clarifies most of the discussion r.e. names, I think.